src/components/layout/Stack/Stack.tsx

```diff
@@ -0,0 +1,32 @@
+import styled, { css } from 'styled-components'
+import { SpaceLength } from '../../../themes/createSpace'
+import { useTheme } from '../../../hooks/useTheme'
+
+/**
+ * @param spaceLength rem の値
+ * @param splitAfter nth-child に渡す値
+ */
+export const Stack = styled.div<{
+  recursive?: boolean
+  spaceLength?: SpaceLength
+  splitAfter?: number
+}>(({ recursive = false, spaceLength = 1, splitAfter }) => {
+  const { space } = useTheme()
+
+  return css`
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    ${!recursive && '>'} * + * {
+      margin-top: ${space(spaceLength)};
```

Comment:
```suggestion
      margin-top: ${space(spaceLength)};
      margin-bottom: 0;
```

`margin-bottom` 持ってる要素が子要素になると `margin-bottom` と `margin-top` が混在して均一な余白が意図しない値になる可能性があるので、リセットしておいたほうが安全かも。